chart/values.yaml

```diff
@@ -0,0 +1,49 @@
+CliVersion: linkerd/cli edge-19.2.5
+ControllerComponentLabel: linkerd.io/control-plane-component
+ControllerImage: gcr.io/linkerd-io/controller:edge-19.2.5
+ControllerLogLevel: info
+ControllerReplicas: 1
+ControllerUID: 2103
+CreatedByAnnotation: linkerd.io/created-by
+DestinationAPIPort: 8086
+EnableH2Upgrade: true
+EnableHA: false
+EnableTLS: false
+GlobalConfig: '{"linkerdNamespace":"linkerd","cniEnabled":false,"identityContext":null}'
+GrafanaImage: gcr.io/linkerd-io/grafana:edge-19.2.5
+GrafanaVolumeName: data
+IgnoreInboundPorts: 4190,4191
+ImagePullPolicy: IfNotPresent
+InboundAcceptKeepaliveMs: 10000
+InboundPort: 4143
+Namespace: linkerd
+NoInitContainer: false
+OutboundConnectKeepaliveMs: 10000
+OutboundPort: 4140
+ProfileSuffixes: .
+PrometheusImage: prom/prometheus:v2.7.1
+PrometheusVolumeName: data
+ProxyAutoInjectEnabled: false
+ProxyConfig: '{"proxyImage":{"imageName":"gcr.io/linkerd-io/proxy:edge-19.2.5","pullPolicy":"IfNotPresent"},"proxyInitImage":{"imageName":"gcr.io/linkerd-io/proxy-init:edge-19.2.5","pullPolicy":"IfNotPresent"},"destinationApiPort":{"port":8086},"controlPort":{"port":4190},"ignoreInboundPorts":[],"ignoreOutboundPorts":[],"inboundPort":{"port":4143},"metricsPort":{"port":4191},"outboundPort":{"port":4140},"resource":{"requestCpu":"","requestMemory":"","limitCpu":"","limitMemory":""},"proxyUid":"2102","logLevel":{"level":"warn,linkerd2_proxy=info"},"disableExternalProfiles":false}'
```

Comment:
Can this come from a file instead? Maybe as YAML that helm renders as json? The ugly json string is hard to reason about and version. I think this'll work:

```
{{ .Values.ProxyConfig | toJson }}
```

Reply:
I added the `_config.json` file that includes the `config.global` and `config.proxy` blocks, which are now `include`d in the ConfigMap template.